daemon/src/taker_cfd.rs

```diff
@@ -200,8 +200,8 @@ where
 
         if !order_to_take.is_safe_to_take(OffsetDateTime::now_utc()) {
             bail!(
-                "The maker's offer appears to be outdated, refusing to take: {:?}",
-                &order_to_take
+                "The maker's offer appears to be outdated, refusing to take offer with id {}",
```

Comment:
If the error message is returned to the taker-ui then the order_id is not very useful here.

Reply:
I think the order-id might be useful for debugging, i.e. if the user runs into the problem often the user might report an order-id back to the maker which might help the maker debug. But I agree, for most users it won't be helpful at all.

I will remove the `order_id`, but note that the maker is not getting any feedback. We could think about a feedback loop back to the maker, where the taker sends a message `SomethingIsFishy`, but that's a different feature. I'll write a ticket for that.

Reply:
https://github.com/itchysats/itchysats/issues/1666